src/test/locale/hi.js

```diff
@@ -4,7 +4,7 @@ import moment from '../../moment';
 localeModule('hi');
 
 test('parse', function (assert) {
-    var tests = 'जनवरी जन._फ़रवरी फ़र._मार्च मार्च_अप्रैल अप्रै._मई मई_जून जून_जुलाई जुल._अगस्त अग._सितम्बर सित._अक्टूबर अक्टू._नवम्बर नव._दिसम्बर दिस.'.split(
+    var tests = 'जनवरी जन._फरवरी फ़र._मार्च मार्च_अप्रैल अप्रै._मई मई_जून जून_जुलाई जुल._अगस्त अग._सितंबर सित._अक्टूबर अक्टू._नवंबर नव._दिसंबर दिस.'.split(
```

Comment:
Why are the tests changed? Will both inflections work or just the new one?

Reply:
Thanks for pointing this out. I have fixed it with the help of regular expressions. Now both the inflections work along with the testcases.

Reply:
If both inflections work, could you add tests for both inflections?

Reply:
@marwahaha  Done. I have added the test using nomination and accusation.

Reply:
Could you add parsing tests for both inflections?

Reply:
@marwahaha I have added the parsing testing for both inflections.